tracer/src/Datadog.Trace/Agent/StatsBuffer.cs

```diff
@@ -57,9 +57,15 @@ public void Reset()
             Start = DateTimeOffset.UtcNow.ToUnixTimeNanoseconds();
         }
 
-        public void Serialize(Stream stream, long bucketDuration)
+        public void Serialize(Stream stream, long bucketDuration, string processTags)
```

Comment:
This doesn't feel like the right place to add a `processTags` parameter. Would it be possible to add process tags to `ClientStatsPayload` and access it from the `_header` field like we do with `Hostname`, `Env`, and `Version`?